python/ray/_private/runtime_env/packaging.py

```diff
@@ -646,8 +646,12 @@ async def download_and_unpack_package(
                         f"Failed to download runtime_env file package {pkg_uri} "
                         "from the GCS to the Ray worker node. The package may "
                         "have prematurely been deleted from the GCS due to a "
-                        "problem with Ray. Try re-running the statement or "
-                        "restarting the Ray cluster."
+                        "long upload time or a problem with Ray. Try setting the "
```

Comment:
Can we distinguish these to cases? I think we can figure out if the failure is due to timeout from `internal_kv_get `?

Reply:
That would be ideal, I'm not sure how to do this though.  We aren't setting set a timeout in `internal_kv_get`, all we know at this point is that the file is not there, so we're giving our best guess in the error message